pkg/util/scrubber/yaml_scrubber.go

```diff
@@ -0,0 +1,83 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package scrubber
+
+import (
+	"gopkg.in/yaml.v2"
```

Comment:
Any reason not to use v3? Follow-up: Any concerns with version and marshaling/unmarshaling in regards to which we pick?

Reply:
We mostly seems to use the V2 in the code (~77 V2 import vs ~7 V3). So I just follow along. Moving the code base to v3 would be for another PR I think.